inc/targetticket.class.php

```diff
@@ -794,20 +697,6 @@ public function pre_purgeItem() {
       return true;
    }
 
-   public function post_addItem() {
-      parent::post_addItem();
-      if ($this->input['show_rule'] != PluginFormcreatorCondition::SHOW_RULE_ALWAYS) {
-         $this->updateConditions($this->input);
-      }
-   }
-
-   public function post_updateItem($history = 1) {
-      parent::post_updateItem();
-      if ($this->input['show_rule'] != PluginFormcreatorCondition::SHOW_RULE_ALWAYS) {
-         $this->updateConditions($this->input);
-      }
-   }
```

Comment:
factorize with all targets types, see implementation in AbstractTarget. 

If adding actors by default is not wanted, then create for each class a method to add default actors.